volatility3/framework/plugins/windows/driverscan.py

```diff
@@ -33,8 +33,9 @@ def get_requirements(cls):
     def scan_drivers(
         cls,
         context: interfaces.context.ContextInterface,
+        kernel_module_name: str,
```

Comment:
If you're handing in the name of the kernel module, why are you also handing in the layer name and the symbol_table_name?  Will these ever be *different* than `kernel.layer_name` and `kernel.symbol_table_name`?  I'm happy to accept there might be instances where these would differ, but if you're not aware of any or can't think of any in the future, it might be worth reducing this (it can always be changed again in the future if it turns out they do need to be different).

Reply:
Fixed